### PR TITLE
Pract.01 Add 01-03 | Velislava Veneva

### PR DESCRIPTION
**Pull request by velislavav**
Velislava Veneva
SI 1 5MI0600243

### CODE REVIEW

File: Seminars/Sem.01/Pract.01/Task01.cpp [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+{
+	return num1 < num2 ? num1 : num2;
+}
+
+int lowerNumByReference(const int& num1, const int& num2)
+{
+	return num1 < num2 ? num1 : num2;
+}
+
+int lowerNumByPointer(int* num1, int* num2)
```
Comment (GeorgiTerziev02) [inline Seminars/Sem.01/Pract.01/Task01.cpp]: const int*

File: Seminars/Sem.01/Pract.01/Task02.cpp [hunk 2]
```diff
@@ -1,38 +1,42 @@
...
-
-    cout << "The number of elements greater than the symbol: " << countSymbol(arr, N, symbol) << endl;
-    
-    delete[] arr;
-    return 0;
-}
+#include <iostream>
+
+
+void inputArr(int* arr, const int& SIZE)
```
Comment (GeorgiTerziev02) [inline Seminars/Sem.01/Pract.01/Task02.cpp]: We do not put & on primitive types

File: Seminars/Sem.01/Pract.01/Task02.cpp [hunk 3]
```diff
@@ -1,38 +1,42 @@
...
+{
+	for (int i = 0; i < SIZE; ++i)
+	{
+		std::cout << "Enter element: ";
+		std::cin >> arr[i];
+	}
+}
+
+int countBiggerSymbols(int* arr, const int& SIZE, const char symbol)
```
Comment (GeorgiTerziev02) [inline Seminars/Sem.01/Pract.01/Task02.cpp]: const int*, because we do not change it, also remove &

File: Seminars/Sem.01/Pract.01/Task02.cpp [hunk 4]
```diff
@@ -1,38 +1,42 @@
...
+{ 
+	int counter = 0;
+	for (int i = 0; i < SIZE; ++i)
+	{
+		if (arr[i] > symbol - '0')
```
Comment (GeorgiTerziev02) [inline Seminars/Sem.01/Pract.01/Task02.cpp]: We do not need - '0' in this task

File: Seminars/Sem.01/Pract.01/Task03.cpp [hunk 5]
```diff
@@ -1,68 +1,78 @@
...
+	{
+		for (int j = 0; j < cols; ++j)
+		{
+			std::cout << "Enter element: ";
+			std::cin >> matrix[i][j];
+		}
+	}
+}
+
+int** transposeMatrix(int** matrix, const int rows, const int cols)
```
Comment (GeorgiTerziev02) [inline Seminars/Sem.01/Pract.01/Task03.cpp]: const int** matrix - because we do not change it

File: Seminars/Sem.01/Pract.01/Task03.cpp [hunk 6]
```diff
@@ -1,68 +1,78 @@
...
+	{
+		for (int j = 0; j < cols; ++j)
+		{
+			transposedMatrix[j][i] =  matrix[i][j];
+		}
+	}
+	return transposedMatrix;
+}
+
+void printMatrix(int** matrix, const int rows, const int cols)
```
Comment (GeorgiTerziev02) [inline Seminars/Sem.01/Pract.01/Task03.cpp]: const int** - because we do not change it

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]
Overall good but you have some minor issues like missing consts, please see the comments.

**Review by GeorgiTerziev02 — APPROVED** [review]

**Comment by Justsvetoslavov** [comment]
Unable to cherry-pick due to error. (is branch deleted or corrupted?)
In any case, kudos 👏

**Pull request closed without merging**